lib/gui/os/window-progress.js

```diff
@@ -46,7 +47,8 @@ exports.currentWindow = electron.remote.getCurrentWindow();
 exports.set = (percentage, status) => {
   exports.currentWindow.setProgressBar(utils.percentageToFloat(percentage));
   const statusString = status === 'check' ? 'Validating' : 'Flashing';
-  exports.currentWindow.setTitle(`Etcher – ${statusString} ${percentage}%`);
+  const dash = '\u2013';
```

Comment:
So this is not shown at all in macOS, given we don't even have a title bar on that platform, but I guess that's "expected".